pegasus/router.rb

```diff
@@ -284,6 +284,11 @@ def update_actionview_assigns
       response.headers['X-Frame-Options'] = 'ALLOWALL'
     end
 
+    if @header['embeddable']
```

Comment:
Just for my own understanding, where is `embeddable` being set?

Reply:
Line 5 of `pegasus/sites.v3/code.org/public/embeddable_state_map.haml`

Reply:
Oh I see! Thank you!